packages/core/lib/commands/exec.js

```diff
@@ -78,6 +78,12 @@ const command = {
         }
         return;
       })
+      .then(compilationOutput => {
+        // save artifacts if compilation took place
+        if (compilationOutput) {
+          return Contracts.save(config, compilationOutput.contracts);
```

Comment:
While we're at it, I think we should make `save` take the entirety of the output of `compile`, not just `contracts`

Reply:
Just out of curiosity, why is this important?

Reply:
I want `save()` eventually to write to Truffle DB, and that will need the compilations data also. Figure everything should get passed to that method, cause who knows what it'll need... it's presumptuous to pre-slice part of the output from `compile()`